Chargebee/Classes/Configuration/Chargebee.swift

```diff
@@ -39,10 +39,20 @@ public class Chargebee {
         CBSubscriptionManager().retrieveSubscription(network: request, logger: logger, handler: handler)
     }
    
+    
+    @available(*, deprecated, message: "This will be removed in upcoming versions, Please use this API func retrieveSubscriptionsList(queryParams: [String: String]? = nil, handler: @escaping RetrieveSubscriptionHandler)")
     public func retrieveSubscriptions(queryParams: [String: String]? = nil, handler: @escaping SubscriptionHandler) {
         let logger = CBLogger(name: "Subscription", action: "Fetch Subscription using customerId")
         logger.info()
+        
+        let request = CBAPIRequest(resource: SubscriptionResource(queryParams: queryParams))
+        CBSubscriptionManager().retrieveSubscriptions(network: request, logger: logger, handler: handler)
+    }
 
+    public func retrieveSubscriptionsList(queryParams: [String: String]? = nil, handler: @escaping RetrieveSubscriptionHandler) {
```

Comment:
Why can we not use `retrieveSubscriptions`?

Reply:
Can you please verify this again?

Reply:
due to some problem in Xcode it didn't reflected in my machine but now its working as expected refactored the name and reverted back with existing name  `retrieveSubscriptions` Thanks !